### PR TITLE
Add in Mac Setup

### PR DESCRIPTION
**Pull request by brandondunne**

### CODE REVIEW

**Comment by brandondunne** [comment]
cc @jrafanie @Fryguy

File: developer_setup.md [hunk 1]
```diff
@@ -34,6 +34,37 @@
       ```
 
 ### Mac
+  1. Install PostgreSQL
+    - Download and install the .dmg for 9.2.4 for MacOSX:  http://www.enterprisedb.com/products-services-training/pgdownload#osx
+    - Open the dmg and run the installer (It will update your shared memory settings and prompt for a reboot, reboot it)
+    - Open the dmg and run the installer again after the reboot.
```
Comment (jrafanie) [inline developer_setup.md]: Does anyone use the dmg way anymore?  @kbrock @Fryguy @matthewd @tenderlove 

I'm using postgresql from homebrew I think.

Comment (brandondunne) [inline developer_setup.md]: @jrafanie I didn't write any of this, not sure what you're supposed to do for Mac.  I'm just consolidating unmaintained doc from other sources. :dragon:

Comment (Fryguy) [inline developer_setup.md]: Yeah the homebrew way is easier to describe.  I can push a change here with updated Mac doco if necessary

Comment (tenderlove) [inline developer_setup.md]: I'm using PG from homebrew as well.

Comment (brandondunne) [inline developer_setup.md]: @Fryguy It would be great if you could add the Mac instructions here or in a follow-up PR, whichever one is easier.

Comment (kbrock) [inline developer_setup.md]: Think we're down to ~3 devs with Mac postgres (not homebrew).

I encourage everyone to go over to homebrew.
All new installs are using homebrew.

**Comment by Fryguy** [comment]
I updated @brandondunne's fork with a PR to update the Mac instructions.  I changed formatting a bit there, so there is an inconsistency.  Once we merge this one, I'll make a separate PR to fix the formatting and other cleanup in the rest of the document.

**Pull request merged**